app.js

```diff
@@ -42,13 +42,11 @@ app.use('/projects', projects);
 app.use('/webhooks', webhooks);
 
 // Pages
-app.use('/pages', express.static(config.deploy.publicPagesDir));
-app.use('/pages/:namespace/:project/*', function(req, res, next) {
-    // Serve directory indexes for public/ftp folder (with icons)
-    var p = req.originalUrl.replace(/^\/pages\//, '');
-    var dir = path.join(config.deploy.publicPagesDir, p);
-    var index = serveIndex(dir, {'icons': true})
```

Comment:
this (according to the original comment) was the "FTP" route that allowed directory contents to be listed.